Documentation/GeneralConventions/ReviewInformation.rst

```diff
@@ -0,0 +1,73 @@
+.. include:: ../Includes.txt
+.. highlight:: rst
+
+.. _add-review-information:
+
+======================
+Add Review Information
+======================
+
+Until we can assure, that the published documentation is generally correct and up-to-date,
+you can add review information to the top of every page.
+
+Here are some examples:
+
+
+Add Review Information to a Page
+================================
+
+Example for Good Content
+------------------------
+
+.. note::
```

Comment:
I'm not sure about the approach to add "correct" to pages. This will just clutter the content and is only valid until the next change is introduced (That's why the concrete version and date is needed to be added).

Also this might give the feeling everything else is not correct.

I would only add the warnings I guess.

Reply:
Yes, that is a good point. My motivation was to try to find an approach for systematically reviewing an entire manual which is currently not entirely up-to-date for a branch (like "Extbase / Fluid book" or "TYPO3 Explained"). If one person reviews entire manual until it is correct, we do not need this. If there are several people doing this, we need some method of tracking what is already been reviewed. Otherwise we have some pages being read again and again while others are sadly lacking. But I suppose the coordination could also be done elsewhere. I will push another change.

Reply:
I understand. We could use the `todo` directive. This can be turned on and off during rendering. So one could, maybe, add todo annotations to all pages. Once a section is checked, this can be removed. This way one can see what sections still need to be checked. For local rendering this can be turned on, while kept off on production.